DataStructures/List.cs

```diff
@@ -0,0 +1,116 @@
+﻿using System;
+
+namespace DataStructures
+{
+    public class List<T>
+    {
+        private T[] items;
+        private int count;
+
+        public List()
+        {
+            items = new T[5];
+            count = 0;
+        }
+        public List(int index)
+        {
+            items = new T[index];
+            count = index;
```

Comment:
As I said before you should receive quantity and not the index, and also `count = index ` is bad, with quantity you say that you want a list that have initially X spaces, but that does not mean that you have X items, at this point you have 0 items.

Reply:
Yes, I forgot it, I will fix it